### ISSUE TITLE
Feature's label is not displayed in result print

### ISSUE THREAD

**Issue by Quanar** [mapfish/mapfish-print#663]
I am using Geoserver 2.10.0 for printing map. Geoserver uses mapfish-print for printing. 
I wanted to print a vector layer with a label in Russian. And in the result of print this label is not displayed, but when I set label in English label is displayed. 
What should I do to solve this problem?

Code example:
..................................................................................................................
"type": 'Vector',
                    "styles": highlighFeaturesStyle,
                    "geoJson": {
                        "type": "FeatureCollection",
                        "features":  [ {
                                  "type": "Feature",
                                  "geometry": JSON.parse(_geojson),
                                  "properties": {
                                        "_style": {
                                              "strokeColor": "#336791",
                                              "strokeOpacity": 0.8,
                                              "strokeWidth": 3,
                                              "fillColor": "#336791",
                                              "fillOpacity": 0.8,
                                              "label": "Текст пример",
                                              "labelYOffset": "14.0",
                                              "fontColor": "#000000",
                                              "fontFamily": "sans-serif",
                                              "fontSize": "10px",
                                              "fontStyle": "normal",
                                              "fontWeight": "bold"
                                            }
                                        }
                                } ]
                         }  
..................................................................................................................

**Comment by Feofilakt** [comment]
Looks like someone haven't been using Unicode sometime. Its interesting if they still support version 2 i.e. fixing bugs.

**Comment by pvalsecc** [comment]
No, we don't work on version 2 anymore. But, usually PRs are welcome and we can do releases for those.

**Comment by Feofilakt** [comment]
Well, here you are :)
https://github.com/mapfish/mapfish-print/pull/986
Though I don't sure if branch 2.1.x is stable, there are some checks and tests errors.

**Comment by pvalsecc** [comment]
Yeah, cicrcleci doesn't work with this branch. But I don't understand why travis didn't trigger...

**Comment by pvalsecc** [comment]
Or @sbrunner, can you have a look?

**Comment by sbrunner** [comment]
@pvalsecc We don't seem to have any CI configured on this branch ...
